specifications/active/aRMT-healthkit-1.0.0.yml

```diff
@@ -0,0 +1,82 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.0.0
+assessment_type: QUESTIONNAIRE
+doc: aRMT Healthkit data definition.
```

Comment:
Can you maybe elaborate more here, that we are using a generic schema for all data types and how it has to be interpreted?